namada/namada-internal-testnet.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
My only comment here is to maybe rename this config to `namada/namada-internal-devnet.json` to match the chain ID.